Model/Message.model.js

```diff
@@ -0,0 +1,33 @@
+const { Schema, model } = require("mongoose")
+
+const messageSchema = new Schema(
+    {
+        destination: {
+            type: String
+        },
+        message: {
+            type: String
+        },
+        number: {
+            type: Number
+        },
+        status: {
+            type: Boolean
```

Comment:
Why do you need status, send, confirmed? 
Do you think that this could be saved in one field?